xcp_d/workflows/anatomical/plotting.py

```diff
@@ -137,19 +137,10 @@ def init_brainsprite_figures_wf(
 
         # Modify template scene file with file paths
         plot_slices = pe.Node(
-            Function(
-                function=plot_slice_for_brainsprite,
-                input_names=[
-                    'nifti',
-                    'lh_wm',
-                    'rh_wm',
-                    'lh_pial',
-                    'rh_pial',
-                ],
-                output_names=['out_files'],
-            ),
+            PlotSlicesForBrainSprite(n_procs=config.nipype.omp_nthreads),
```

Comment:
I checked the brainsprites from the integration tests and they looked good, so I don't think there's an issue with multiprocessing shuffling the slices.